examples/kubernetes/kernel-check/kernel-check.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: bpf-kernel-check
+  name: bpf-kernel-check
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: bpf-kernel-check
+  template:
+    metadata:
+      labels:
+        app: bpf-kernel-check
+    spec:
+      nodeSelector:
+      # run deployment on specific node, e. g. nodename:cilium
+      containers:
+        - name: cilium
+          image: docker.io/cilium/cilium:stable
+          command: ["sh", "-c", "cilium kernel-check"]
+          resources:
```

Comment:
I think we can remove `resources` entirely.

Reply:
thx, you are right: removed.